Users/Tutorial.rst

```diff
@@ -226,7 +226,7 @@ sequentially.
 .. note::
 
     For a list of configuration options for the bears, take a look at our
-    `bear-docs <https://github.com/coala/bear-docs>`_ documentation.
+    `Coala Bears <https://coala.io/#/languages>`_ documentation.
```

Comment:
coala is always written with a lowercase c.

Reply:
`... our <coala languages> directory.` ?

Reply:
done